.github/workflows/build.yml

```diff
@@ -78,8 +78,9 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: Install dependencies if needed.
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
-        run: yarn install --frozen-lockfile
+        run: |
```

Comment:
Why did you need to remove the cache-hit? Maybe you need to include the new lock file hash in the key instead? (line 76)

Reply:
great catch.  I'm not an expert with github workflows, but I'll try to make that change and see if it works.